storage/auth_store.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 Nippon Telegraph and Telephone Corporation.
```

Comment:
let's move this out of etcd/stroage to etcd/auth. (all you need is the backend not KV, we will probably rename storage pkg to something better in the future)